src/formio/components/TextField.js

```diff
@@ -26,7 +26,7 @@ class TextField extends Formio.Components.components.textfield {
     return info;
   }
 
-  checkComponentValidity(data, dirty, row, options = {}){
+  checkComponentValidity(data, dirty, row, options = {}) {
```

Comment:
Revert this, this has nothing to do with the actual bug or bug fix. See earlier remarks about prettier.